src/styles/blocks/header.scss

```diff
@@ -38,5 +38,10 @@
     text-transform: uppercase;
     font-size: 12px;
     font-weight: 500;
+
+    &:hover {
+      color: #00acdc;
+      transition-duration: $duration;
```

Comment:
The SCSS variable `$duration` is not defined in this file or in the code provided. Make sure to define this variable in a proper scope, or replace it with an actual value.